### PR TITLE
k8s: Add k8s test for kubectl cp

### LINKED ISSUE #1968: Add kubectl cp test

### PR DESCRIPTION
**Pull request by GabyCT**
This will copy a file from the host to a pod.

Fixes #1968

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: integration/kubernetes/k8s-copy-file.bats [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+@test "Copy file in a pod" {
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-env.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Create a file
+	echo "Hello" > file.txt
```
Comment (chavafg) [inline integration/kubernetes/k8s-copy-file.bats]: `file.txt` could be a variable

Comment (GabyCT) [reply integration/kubernetes/k8s-copy-file.bats]: @chavafg changes applied

File: integration/kubernetes/k8s-copy-file.bats [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+@test "Copy file in a pod" {
+	# Create pod
+	kubectl create -f "${pod_config_dir}/pod-env.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Create a file
+	echo "Hello" > "$file_name"
```
Comment (devimc) [inline integration/kubernetes/k8s-copy-file.bats]: `"Hello"` can be a variable too and use it later to check file's content

Comment (GabyCT) [reply integration/kubernetes/k8s-copy-file.bats]: @devimc changes applied

**Pull request merged**